code/modules/mob/living/simple_animal/friendly/corgi.dm

```diff
@@ -798,6 +798,17 @@
 			if(Adjacent(victim) && IsVictim(victim)) //Seriously don't try to rescue the dead.
 				rescue(victim)
 
+/mob/living/simple_animal/corgi/cultify()
+	if(prob(5))
```

Comment:
Seems kind of pointless with it being so low and since cultify so rare why even have it be RNG?

Reply:
Was having it be a nat 20 required to get a Corgerus, but okay